docs/4.3/kubernetes_ssh.md

```diff
@@ -34,40 +30,74 @@ proxy_service:
-  a load balancer, this must be the load balancer's public address.
+- `public_addr` defines the publicly accessible address which Kubernetes API clients
+  like `kubectl` will connect to. This address will be placed inside of kubeconfig on
+  a client's machine when a client executes tsh login command to retrieve its certificate.
+  If you intend to run multiple Teleport proxies behind a load balancer, this must
+  be the load balancer's public address.
 
-* `listen_addr` defines which network interface and port the Teleport proxy server
+- `listen_addr` defines which network interface and port the Teleport proxy server
   should bind to. It defaults to port 3026 on all NICs.
 
-## Connecting the Teleport proxy to Kubernetes
+### Connecting the Teleport proxy to Kubernetes
+
+There are two options for setting up Teleport to access Kubernetes:
+
+## Deploy Inside Kubernetes as a pod
+
+Deploy Teleport Proxy service as a Kubernetes pod inside the Kubernetes cluster you
+want the proxy to have access to inside the Kubernetes cluster you want the proxy
+to have access to.
+
+```yaml
+# snippet from /etc/teleport.yaml on the Teleport proxy service:
+proxy_service:
+    # create the 'kubernetes' section and set 'enabled' to 'yes':
+    kubernetes:
+        enabled: yes
+```
+
+If you're using Helm, we've a chart that you can use. Run these commands:
 
-There are two ways this can be done:
+```bash
+$ helm repo add gravitational https://charts.gravitational.io
+$ helm install teleport gravitational/teleport
+```
+You will still need a correctly configured `values.yaml` file for this to work. See
+our [Helm Docs](https://github.com/gravitational/teleport/tree/master/examples/chart/teleport#introduction) for more information.
 
-1. Deploy Teleport Proxy service as a Kubernetes pod inside the Kubernetes cluster you want the proxy to have access to.
-   No Teleport configuration changes are required in this case.
-2. Deploy the Teleport proxy service outside of Kubernetes and update the Teleport Proxy configuration with Kubernetes
-   credentials. In this case, we need to update `/etc/teleport.yaml` for the proxy service as shown below:
+![teleport-kubernetes-inside](img/teleport-k8s-pod.svg)
+
+## Deploy Outside of Kubernetes
+
+Deploy the Teleport proxy service outside of Kubernetes and update the Teleport
+Proxy configuration with Kubernetes credentials. and update the Teleport Proxy
+configuration with Kubernetes credentials. In this case, we need to update `/etc/teleport.yaml`
+for the proxy service as shown below:
 
 ```yaml
-# snippet from /etc/teleport.yaml on the proxy service deployed outside k8s:
+# snippet from /etc/teleport.yaml on the Teleport proxy service:
 proxy_service:
+  # create the 'kubernetes' section and set 'enabled' to 'yes':
   kubernetes:
-    kubeconfig_file: /path/to/kubeconfig
+    enabled: yes
+    public_addr: [teleport.example.com:3026]
+    listen_addr: 0.0.0.0:3026
```

Comment:
optional: add a comment explaining the difference here:
```
# The address for the proxy process to accept k8s requests.
listen_addr: 0.0.0.0:3026
# The address used by the clients after tsh login. If you run a load balancer in front of this proxy, use the address of that balancer here. Otherwise, use the address of the host running this proxy.
public_addr: [teleport.example.com:3026]

Reply:
I think this is helpful, I've added it.